Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/ICursor.cs

```diff
@@ -1,11 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.InputSystem.InputHandlers;
-using Microsoft.MixedReality.Toolkit.InputSystem.Pointers;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.InputSystem.Cursors
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem
```

Comment:
MRTK Interface names should be prefixed with either IMixedReality or IMixedRealityToolkit

Reply:
done